src/Plots.jl

```diff
@@ -116,7 +116,7 @@ ignorenan_minimum(x) = Base.minimum(x)
 ignorenan_maximum(x::AbstractArray{F}) where {F<:AbstractFloat} = NaNMath.maximum(x)
 ignorenan_maximum(x) = Base.maximum(x)
 ignorenan_mean(x::AbstractArray{F}) where {F<:AbstractFloat} = NaNMath.mean(x)
-ignorenan_mean(x) = Base.mean(x)
```

Comment:
Maybe `Statistics.mean` to be explicit?

Reply:
Makes sense. Applied in 6d0d8ea90aefb3044c3d9778cfb6e7ff2c5ef11a